django_irods

```diff
@@ -1 +1 @@
-Subproject commit 5a0359d72f2b4ec3a43e38a2be22d9ed12f092b0
```

Comment:
@ironfroggy looks like your django_irods submodule is not up to date with the version on develop, you may need to update this submodule to the latest commit as in develop to avoid merging issues when this branch is merge-ready.